libbeat/docs/step-configure-credentials.asciidoc

```diff
@@ -0,0 +1,29 @@
+ifeval::["{beatname_lc}"!="winlogbeat"]
+. If you've secured Elasticsearch and Kibana, you need to pass credentials when
+you run the commands that set up and start {beatname_uc}. You can either pass
+credentials at the command line (as shown later in the getting started guide),
+or specify credentials in the config file. If you plan to run {beatname_uc} as a
+service, you must specify credentials in the config file.
+endif::[]
+ifeval::["{beatname_lc}"=="winlogbeat"]
+. If you've secured Elasticsearch and Kibana, you need to specify credentials
+in the config file before you run the commands that set up and start
+{beatname_uc}.
+endif::[]
++
+For example:
++
+[source,yaml]
+----
+output.elasticsearch:
+  hosts: ["myEShost:9200"]
+  username: elastic
+  password: elastic
+setup.kibana:
+  host: "mykibanahost:5601"
+  username: elastic
+  password: elastic
```

Comment:
I've made a note of this behavior and will update the docs before beta2. For beta1 users, might be better to leave this as it is for now.